### ISSUE TITLE
[Feature request] Create a `.service` file in `/usr/share/dbus-1/services/`

### ISSUE THREAD

**Issue by zefr0x** [lxqt/pcmanfm-qt#1834]
**Is your feature request related to a problem? Please describe.**
When PCManFM-Qt is not running as a daemon there will not be a dbus interface for applications to open directories using PCManFM-Qt.

**Describe the solution you'd like**
Create a `.service` file in `/usr/share/dbus-1/services/` so if any applications sent a request to the dbus interface it will automatically start the PCManFM-Qt daemon to handle the request.

For example the Thunar file manager has `/usr/share/dbus-1/services/org.xfce.Thunar.FileManager1.service`:
```
[D-BUS Service]
Name=org.freedesktop.FileManager1
Exec=/usr/bin/Thunar --gapplication-service
SystemdService=thunar.service
```

For PCManFM-Qt it would be like in `/usr/share/dbus-1/services/org.lxqt.PCManFMQt.FileManager1.service`:
```
[D-BUS Service]
Name=org.freedesktop.FileManager1
Exec=/usr/bin/pcmanfm-qt --daemon-mode
```

**Comment by izmyname** [comment]
I know it's old, but it would be a nice feature, paired with stuff like mount handling.

**Comment by ahsand97** [comment]
I think we should provide those service files, normally you'd use pcmanfm-qt along with the desktop module so it's always running in the background and the d-bus service is registered, but, if you're not using pcmanfmqt's desktop module then the d-bus services will not be registered and that will cause the d-bus `org.freedesktop.FileManager1` methods call fail. @tsujan Any toughts on this?

**Comment by tsujan** [comment]
I disagree with the whole idea. If pcmanfm-qt isn't running as a daemon or for desktop, it means that it's not chosen by the user, whether inside an LXQt session or outside it. LXQt is modular.

**Comment by tsujan** [comment]
PCManFM-Qt has a daemon mode and a desktop mode. Understandably, we start it in the desktop mode with an LXQt session by default, but the user can disable it. Also, the user is free to auto-start it in one of those modes anywhere.

LXQt doesn't enforce such things on users, regardless of what other DEs may do. Hence closing this.
